attributes/visibility/form.php

```diff
@@ -2,41 +2,65 @@
 defined('C5_EXECUTE') or die('Access Denied.');
 /** @var \Concrete\Core\Form\Service\Form $form */
 /** @var \Concrete\Core\User\Group\Group[] $visibleGroups */
+/* @var Concrete\Core\Attribute\View $view */
 $visibleGroups = $visibleGroups ?? [];
 /** @var \Concrete\Core\User\Group\Group[] $optionGroups */
 $optionGroups = $optionGroups ?? [];
 $allowMultiple = $allowMultiple ?? false;
 $akID = $akID ?? null;
+$isV9 = version_compare(app('config')->get('concrete.version'), '9.0.0', '>=');
 ?>
 <div class="form-group">
-    <?php
+<?php if ($isV9 && $allowMultiple) {
+    foreach ($optionGroups as $option) {
+        /** @var \Concrete\Core\User\Group\Group $option */ ?>
+            <div class="form-check">
+                <?=$form->checkbox($view->field('visibleGroups') . '[]', $option['id'], in_array($option['id'], $visibleGroups)); ?>
+                <label class="form-check-label" for="<?= $view->field('atSelectOptionValue') . '_' . $option['id']; ?>">
+                    <?=$option['label']?>
```

Comment:
### Better to have
Let's use `<?= htmlspecialchars($option['label'], ENT_QUOTES, APP_CHARSET); ?>` to enhance security